amundsen_application/static/js/ducks/bookmark/reducer.ts

```diff
@@ -33,11 +41,25 @@ export function removeBookmark(resourceKey: string, resourceType: string): Remov
     type: RemoveBookmark.REQUEST,
   }
 }
+export function removeBookmarkFailure(): RemoveBookmarkResponse {
+  return { type: RemoveBookmark.FAILURE };
+}
+export function removeBookmarkSuccess(resourceKey: string, resourceType: string): RemoveBookmarkResponse {
+  return { type: RemoveBookmark.SUCCESS, payload: { resourceKey, resourceType } };
+}
+
 export function getBookmarks(): GetBookmarksRequest {
   return {
     type: GetBookmarks.REQUEST,
   }
 }
+export function getBookmarksFailure(): GetBookmarksResponse {
+  return { type: GetBookmarks.FAILURE };
```

Comment:
Why not return a `payload: { bookmarks: [] }` here?

Reply:
We didn't use the value at all in the reducer so I deleted it. Seems like if we included it in the payload it should have been intentionally used in the reducer. Let me know what your intent was when writing this and I'll update accordingly.

Reply:
The bookmarks _should_ behave like other reducers and set the value to default/empty array. If there is a discrepancy it's probably just an oversight or changed pattern that wasn't updated. 


That being said, I do think we need to revisit failure scenarios from a holistic perspective. Our current pattern is to fail quietly, but perhaps we should surface some error message to the user or do an automatic retry? I'm not sure what the best approach is, but we should standardize it. (Out of scope)

Reply:
There's an ancient ticket to revisit our error handling. We've been making these decisions on a case by case basis, but some guidelines would be nice. I will update the failure payload back to `{ bookmarks: [] }`.